### ISSUE TITLE
有关Adapative_label_clusters.py的问题

### ISSUE THREAD

**Issue by htt6** [huiyegit/APLC_XLNet#1]
Adapative_label_clusters.py中186-188行     
multiplier = (cluster_onehot == 0).long()
cluster_root_loss = cluster_root_loss * multiplier.float()
这里为什么选择cluster_onehot == 0，感觉应该是==1，还有最后计算loss的方法和论文中公式是不是不一致？

**Comment by huiyegit** [comment]
I show a simple example to explain it. Assume that there are 12 labels in the label set. Li denotes i-th label. Pi denotes the parent note of the i-th tail cluster. One training sample has two positive labels, label 1 and label 4. The labels whose losses to be computed are in red in the figure below.  In the implementation, the loss of the parent node p2 is computed, while the equation in the paper does not include it. The source code you mentioned is to compute the loss of this parent node p2.
![pdfresizer com-pdf-crop (2)-page-001](https://user-images.githubusercontent.com/31486250/90342467-870d1b80-dfd6-11ea-9dc7-35e934d54ba0.jpg)
